cinema/views.py

```diff
@@ -1,8 +1,10 @@
 from datetime import datetime
 
 from django.db.models import F, Count
-from rest_framework import viewsets
+from rest_framework import viewsets, mixins
```

Comment:
Consider using `ModelViewSet` instead of `GenericViewSet` with mixins for `GenreViewSet`, `ActorViewSet`, and `CinemaHallViewSet`. This would simplify the code since `ModelViewSet` already includes the `ListModelMixin` and `CreateModelMixin` by default.